bitnami/fluent-bit/templates/_helpers.tpl

```diff
@@ -0,0 +1,59 @@
+{{/* vim: set filetype=mustache: */}}
+
+
+{{/*
+Return the proper fluent-bit image name
+*/}}
+{{- define "fluentBit.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Create the name of the deployment
+*/}}
+{{- define "fluentBit.fullname" -}}
+    {{ printf "%s" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "fluentBit.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{ default (include "fluentBit.fullname" .) .Values.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "fluentBit.imagePullSecrets" -}}
+{{- include "common.images.pullSecrets" (dict "images" (list .Values.image) "global" .Values.global) -}}
+{{- end -}}
+
+{{/*
+Return the k8s deploy kind
+*/}}
+{{- define "fluentBit.kind" -}}
+    {{- if .Values.useDaemonset -}}
+        {{ printf "%s" "DaemonSet" }}
+    {{- else -}}
+        {{ printf "%s" "Deployment" }}
+    {{- end -}}
+{{- end -}}
+
+{{/*
+Return the ingress anotation
+*/}}
+{{- define "fluentBit.ingress.annotations" -}}
+{{ .Values.ingress.annotations | toYaml }}
+{{- end -}}
+
```

Comment:
We've used common.tplvalues.render for annotations, without the need of an external helper